generator/python/build.gradle

```diff
@@ -58,15 +65,21 @@ task generateClient(type: GradleBuild) {
                 gitUserId='criteo'
                 gitRepoId='criteo-api-python-sdk'
                 configOptions = [
-                        apiVersion        : parsedJson.info.version,
-                        packageName            : 'criteo-api-sdk',
-                        invokerPackage         : 'Criteo\\SDK',
-                        packageUrl             : 'https://github.com/criteo/criteo-api-python-sdk',
-                        hideGenerationTimestamp: 'true'
+                        apiVersion              : parsedJson.info.version,
+                        packageName             : "${packageBase}".toString(),
```

Comment:
Something is bothering me here.
If I understood well the package name would be for example `com.criteo.api.marketingsolutions.v2021_10`. 
In python, such name would mean that `com` is a package (= folder with a __init__.py file) containing another package called `criteo`, containing another package called `api` and so on util package `v2021_10` that will actually contain the generate SDK. 
In addition the `com.` keycord seems very specific to the java SDKs, and is not used in the python world.
I think that for python it would be better to have one single package called for example `criteo_api_marketingsolutions_v2021_10`.